### PR TITLE
Manual entry time update fix

### LINKED ISSUE #54: [Bug] Manual Entry does not update time listened
**Steps to reproduce:**

1. Go the stats screen
2. Click on a date in the calendar
3. Manually enter a time in minutes meditated i.e. 22
4. Click submit
5. Notice how the time listened does not update to include the 22 minutes on the stats screen

**Expected behaviour:**

Time listened updates with manually entered time

**Actual behaviour:**

Time listened stat does not update and stays at zero or whatever time you already had.

**Smartphone (please complete the following information):**
 - Device/OS: iPhone 12 Simulator

**Screenshots/Recording:**
Screen Shot 2021-08-22 at 1 27 38 PM

simulator_screenshot_3BB076CD-F8AF-4680-8D57-5B3EABCD787D

simulator_screenshot_FC93E4C3-A3F6-4466-9881-FC679326A2D2

### PR DESCRIPTION
**Pull request by alexandrvicente**
## Description

Adds the missing conversion from minutes to ms to the manual entry modal and comments explaining deletion and calendar date conversion.

## Ticket Link

Closes #54

## How has this been tested?

iPhone X 14.7.1

## Screenshots

Adding
Updated time

## Checklist

- [x] Added a "Closes [issue number]" in the ticket link section
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens

### CODE REVIEW

**Review by watadarkstar — DISMISSED** [review]
LGTM

**Comment by watadarkstar** [comment]
@alexandrvicente Unit tests are failing, can you fix them

File: constants/Units.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const msPerMinute = 60000
```
Comment (watadarkstar) [inline constants/Units.ts:1]: Nit:

```suggestion
export const MS_PER_MINUTE = 60000
```

Comment (alexandrvicente) [reply constants/Units.ts]: Done

**Review by watadarkstar — APPROVED** [review]

**Pull request merged**